velox/functions/prestosql/JsonFunctions.h

```diff
@@ -19,6 +19,21 @@
 
 namespace facebook::velox::functions {
 
+// is_json_scalar(json) -> boolean
+// Determines if json is a scalar (i.e. a JSON number, a JSON string, true,
+// false or null)
+template <typename T>
+struct isJsonScalarFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      bool& result,
+      const arg_type<Varchar>& json) {
+    auto parsedJson = folly::parseJson(json);
+    result = parsedJson.isNumber() or parsedJson.isString() or parsedJson.isBool() or parsedJson.isNull();
```

Comment:
Let's use `||` instead `or` here to be more consistent with other code in Velox.

Reply:
Thanks, changed accordingly